packages/application-shell-connectors/package.json

```diff
@@ -38,23 +38,23 @@
     "build:typings": "cross-env tsc -p tsconfig.declarations.json --emitDeclarationOnly --declarationDir dist/typings"
   },
   "dependencies": {
-    "@babel/runtime-corejs3": "7.5.5",
+    "@babel/runtime-corejs3": "7.6.0",
     "@commercetools-frontend/constants": "15.0.0",
     "@commercetools-frontend/sentry": "15.0.0",
     "apollo-cache-inmemory": "1.6.3",
     "apollo-client": "2.6.4",
-    "apollo-link-http": "1.5.15",
-    "graphql": "14.4.0",
+    "apollo-link-http": "1.5.16",
+    "graphql": "14.5.4",
     "graphql-tag": "^2.10.1",
     "lodash": "4.17.15",
     "lodash-es": "4.17.15",
     "moment-timezone": "^0.5.26",
     "prop-types": "15.7.2"
   },
   "devDependencies": {
-    "@apollo/react-testing": "3.0.1",
+    "@apollo/react-testing": "3.1.0",
     "react": "16.9.0",
-    "react-apollo": "3.0.1"
+    "react-apollo": "3.1.0"
```

Comment:
Note that the behavior of the data prop being undefined over an empty can be considered breaking maybe. Please check the changelog, usages here and then the apps.